test/jasmine/tests/gl_plot_interact_test.js

```diff
@@ -761,20 +763,18 @@ describe('gl2d interaction', function() {
     it('data-referenced annotations should update on drag', function(done) {
 
         function drag(start, end) {
-            var opts = { buttons: 1 };
-
-            mouseEvent('mousemove', start[0], start[1], opts);
-            mouseEvent('mousedown', start[0], start[1], opts);
-            mouseEvent('mousemove', end[0], end[1], opts);
-            mouseEvent('mouseup', end[0], end[1], opts);
+            mouseEvent('mousemove', start[0], start[1]);
+            mouseEvent('mousedown', start[0], start[1], { buttons: 1 });
+            mouseEvent('mousemove', end[0], end[1], { buttons: 1 });
+            mouseEvent('mouseup', end[0], end[1]);
```

Comment:
https://github.com/plotly/plotly.js/pull/1301/commits/5f36071a2a74e0ded7400896f24d3ab22e5aa3c1 broke the test w/o me noticing. As test cases that require a gl context are skipped on CI, this went unnoticed until I now.